docs/chaosiq-on-docker/run-verification.md

```diff
@@ -0,0 +1,54 @@
+## Create a Verification
+
+Verifications can also run on the Docker image. First you will need to create a Verification. Select Verifications in the ChaosIQ Sidebar, and then select the Create a New Verification button.  Select an objective to link the Verification, once an Object has been selected you should see:
+
+![Create Verification][CreateVerification]
+
+Select a Frequency and a Duration, for that sake of running a quick Verification, use something like a Frequency of something like 5 seconds and a Duration of 30 seconds. For the purpose of this, leave the No Condition option set and use the default Verification name, then select the  Create Verification and View Execution Steps button, you will be take to the Run Verification page:
+
+![Run Verification][RunVerification]
+
+
+From the Run Verification page, you can Copy the verify command and extract the URL from the command, this can be used in the Run Verification section.
+
+## Run Verification
+
+A Verification runs using a different runtime pattern from an experiment, it will also populate the Insights view on ChaosIQ, the [Run and Verify page][RunAndVerify] details the differences between *run* and *verify*.
+
+You can then run the Verification with the following command:
+
+```bash
+docker run  -v `pwd`/ctk:/home/svc  -v `pwd`/ctk_config:/tmp/settings \
+ chaosiq/chaostoolkit --settings /tmp/settings/settings.yaml verify  \
```

Comment:
great with settings !! 👍

Reply:
but as a user , where do I get this settings file ? maybe you could add a section to indicate how to login to chaosiq via the command line, where to find that settings file (mainly in the user home folder), etc...
the user needs to signin, select an org and a team, to have its settings file ready for use.

Reply:
damn, this is a next file ... I shall not review in GH as it's not ordered ...